torch/__init__.py

```diff
@@ -1752,6 +1753,108 @@ def registerOp(cls, op_key, full_schema, op_impl, dispatch_key):
             return cls.ops_table[(op_key, dispatch_key)]
 
 
+################################################################################
+# Exposing torch.export() and related support classes
+################################################################################
+
+# TODO(ycao): Move ExportedProgram, Constraint, dynamic_dim etc to torch.compiler namespace
+
+def export(
+    f: Callable,
+    args: Tuple[Any],
+    kwargs: Optional[Dict[str, Any]] = None,
+    *,
+    constraints: Optional[List["torch._dynamo.eval_frame.Constraint"]] = None,
```

Comment:
Did you leave out the return types to be private because moving them all at once will create a giant PR?

Reply:
Yes